src/components/map/MapDetail.js

```diff
@@ -135,16 +136,18 @@ const MapDetail = () => {
       </Container>
       <Routes>
         <Route
-          path="/point"
+          path="/invitation"
           element={
-            <Modal size="big">포토를 가져와서 보여주는 모달입니다.</Modal>
+            <Modal size="small">
+              <NewInvitation />
+            </Modal>
           }
         />
         <Route
-          path="/invitation"
+          path="photos"
```

Comment:
```suggestion
          path="/photos"
```
어디서는 url 앞에 슬래쉬 사용하고 어디서는 url 앞에 슬래쉬를 사용하지 않고 있어서
일관성 있게 하기 위해서 어떻게 할지 정하면 좋을 것 같습니다-!

Reply:
함께 이야기해보시죠!